src/pages/OfferNew/Perils/types.ts

```diff
@@ -8,3 +8,25 @@ export interface Peril {
   info?: string
   icon: JSX.Element
 }
+
+type IconVariant = {
+  svgUrl: string
+}
+
+type IconVariants = {
+  dark: IconVariant
+  light: IconVariant
+}
+
+export type PerilIcon = {
+  variants: IconVariants
+}
+
+export interface PerilV2 {
```

Comment:
do we need `V2`? I think we could kill off the old ones right? And just make this Peril type the _the_ Peril? Assuming we can't use the peril types from graphql, that is